interactive/interactive_client.go

```diff
@@ -99,10 +96,9 @@ func (c *InteractiveClient) InteractivePrompt() {
 		// close up the SIGINT channel so that the receiver goroutine can quit
 		signal.Stop(interruptSignalChannel)
 		close(interruptSignalChannel)
-		// close the underlying client if it exists
-		if client := c.client(); client != nil {
-			client.Close()
-		}
+
+		// init data cleanup
```

Comment:
need a better comment here - this is important - this cleanup shuts the service
```
// cleanup the init data to ensure any services we started are stopped
```

Reply:
👍